src/config.js

```diff
@@ -1,16 +1,16 @@
 export const config = {
-    RPC_URL: 'https://rpc.flixnet.omniflix.network',
-    REST_URL: 'https://rest.flixnet.omniflix.network',
-    EXPLORER_URL: 'https://explorer.omniflix.network',
-    STAKING_URL: 'https://flix.omniflix.co/stake',
-    NETWORK_NAME: 'OmniFlix',
+    RPC_URL: 'wss://<rpc_celestia_node_address>',
+    REST_URL: 'https://<rest_celestia_node_address>',
+    EXPLORER_URL: 'https://celestia.observer/',
+    STAKING_URL: 'https://staking.celestia.observer/',
+    NETWORK_NAME: 'Celestia',
     NETWORK_TYPE: 'testnet',
-    CHAIN_ID: 'flixnet-3',
-    CHAIN_NAME: 'OmniFlix FlixNet-3',
-    COIN_DENOM: 'FLIX',
-    COIN_MINIMAL_DENOM: 'uflix',
-    COIN_DECIMALS: 6,
-    PREFIX: 'omniflix',
+    CHAIN_ID: 'devnet-2',
+    CHAIN_NAME: 'Celestia devnet-2',
+    COIN_DENOM: 'celes',
+    COIN_MINIMAL_DENOM: 'celes',
```

Comment:
BTW, COIN_DENOM and COIN_DENOM usually are not the same. It's only because we screwed up the denom at launch of devnet. Might be good to track this in an issue / TODO to not forget to update this with the next testnet.

Reply:
We already have an issue for it https://github.com/celestiaorg/insync/issues/4 
I will merge this PR. Then, when the celes/uceles is handled, will create a separate PR.
Thanks for mentioning this out.
For the capitalization thing, I think it is related to what we have in the app. Thus, whatever we define there, it will need to be the same here, I guess.